src/components/buttons/Send/SendFlow.tsx

```diff
@@ -197,6 +199,7 @@ const SendFlow = ({ isMobile, onClose }: SendFlowProps) => {
    };
 
    const handlePaymentRequestInput = async (request: string) => {
+      setState(state => ({ ...state, isProcessing: true }));
```

Comment:
are you sure it is needed here because there is no async stuff happening here?

Reply:
good point